docs/modules/plugins/pages/plugin-winrm.adoc

```diff
@@ -0,0 +1,106 @@
+= WinRM Plugin
+
+The plugin provides functionality to execute commands via https://docs.microsoft.com/en-us/windows/win32/winrm/portal[WinRM].
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-winrm', version: '{current-version}')
+----
+
+== Properties
+
+It's allowed to configure unlimited number of WinRM connections via mechanism of the dynamic properties. The properties prefix example is:
+```properties
+winrm.server.my-server
+```
+where `my-server` is a key used to refer WinRm connection in the steps. The key is defined by users, must be unique and can't contain dots.
+
+NOTE: The properties marked with *bold* are mandatory.
+
+[cols="3,1,1,3", options="header"]
+|===
+|Property Name
+|Acceptable values
+|Default
+|Description
+
+|[subs=+quotes]`*winrm.server.<key>.address*`
+|<string>
+|
+|Either full endpoint URL (e.g. `https://10.240.1.1:5986/wsman`) or hostname and port separated by colon (e.g. `10.240.1.1:5986`)
+
+|[subs=+quotes]`*winrm.server.<key>.username*`
+|<string>
+|
+|The name of the user logging in
+
+|[subs=+quotes]`*winrm.server.<key>.password*`
+|<string>
+|
+|The password of the user logging in
+
+
+|`winrm.server.<key>.authentication-scheme`
+a|`Basic` +
+`Digest` +
+`NTLM` +
+`Negotiate` +
+`Kerberos` +
+`CredSSP`
+|`NTLM`
+|Authentication scheme
+
+|`winrm.server.<key>.disable-certificate-checks`
+a|`true` +
+`false`
+|`false`
+|Disable/enable HTTPS certificates validation
+
+|===
+
+== Steps
+
+=== Execute batch command
+
+Executes a native Windows command. A new session is created on the destination host for each step invocation.
```

Comment:
native Windows commands - means win32 built in commands?

Reply:
<img width="1045" alt="Screenshot 2022-08-09 at 08 49 10" src="https://user-images.githubusercontent.com/5081226/183574285-753206d3-6495-433d-af59-3cb3b114569a.png">